packages/scandipwa/src/component/GroupedProductList/GroupedProductList.component.js

```diff
@@ -41,7 +41,7 @@ export class GroupedProductList extends PureComponent {
 
         return (
             <ul>
-                { items.map(({ product, product: { id }, qty }) => (
+                { items.filter(({ product }) => product).map(({ product, product: { id } = {}, qty }) => (
```

Comment:
Sounds strange, maybe worth fixing on products processor level?

Reply:
Done this in scandipwa/catalog-graphql/pull/92